src/chains/config/polygon.config.ts

```diff
@@ -4,7 +4,7 @@ import { TOKENS } from '../../config/tokens.config';
 import { GasPrices } from '../../gas/interfaces/gas-prices.interface';
 import { getCurveVaultTokenBalance } from '../../protocols/strategies/convex.strategy';
 import { VaultDefinition } from '../../vaults/interfaces/vault-definition.interface';
-import { maticTokensConfig } from '../../tokens/config/matic-tokens.config';
+import { maticTokensConfig } from '../../tokens/config/polygon-tokens.config';
```

Comment:
this fine being maticTokensConfig instead of polygon?

Reply:
yes, lol its a process. there is much more matic around still